client/my-sites/hosting/main.js

```diff
@@ -20,31 +20,21 @@ import canSiteViewAtomicHosting from 'state/selectors/can-site-view-atomic-hosti
 import SFTPCard from './sftp-card';
 import PhpMyAdminCard from './phpmyadmin-card';
 import SupportCard from './support-card';
+import PhpVersionCard from './php-version-card';
 import { isEnabled } from 'config';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
-const Hosting = ( { translate, isDisabled, canViewAtomicHosting } ) => {
+const Hosting = ( { translate, isDisabled, canViewAtomicHosting, siteId } ) => {
 	if ( ! canViewAtomicHosting ) {
 		return null;
 	}
 
-	const sftpPhpMyAdminFeatures = isEnabled( 'hosting/sftp-phpmyadmin' ) ? (
-		<>
-			<div className="hosting__layout">
-				<div className="hosting__layout-col">
-					<SFTPCard disabled={ isDisabled } />
-					<PhpMyAdminCard disabled={ isDisabled } />
-				</div>
-				<div className="hosting__layout-col">
-					<SupportCard />
-				</div>
-			</div>
-		</>
-	) : null;
+	const sftpPhpMyAdminFeaturesEnabled = true;
```

Comment:
Ah this was why `DISABLE_FEATURES=hosting/sftp-phpmyadmin npm start` didn't work. :D I can push an update